sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -3,23 +3,13 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 from __future__ import unicode_literals
+from multiprocessing import Event
```

Comment:
Is Event used anywhere?

Reply:
ah it is not - Good catch! vscode must've done the auto import thing 🤦